test/Nancy.Tests/Unit/Sessions/DefaultSessionObjectFormatterFixture.cs

```diff
@@ -68,11 +72,16 @@ public void Should_return_null_when_deserializing_empty_string()
         [Serializable]
         public class Payload : IEquatable<Payload>
         {
-            public int IntValue { get; private set; }
+            public int IntValue { get;  set; }
+
+            public bool BoolValue { get;  set; }
 
-            public bool BoolValue { get; private set; }
+            public string StringValue { get;  set; }
 
-            public string StringValue { get; private set; }
+            public Payload()

```

Comment:
I think this brings the serialization in line with every other serializer, the need for a parameterless ctor is almost expected now-a-days with everything using reflection.